doc/examples/segmentation/plot_label.py

```diff
@@ -39,19 +38,17 @@
 label_image = label(cleared)
 image_label_overlay = label2rgb(label_image, image=image)
 
-fig, ax = plt.subplots(ncols=1, nrows=1, figsize=(6, 6))
+fig, ax = plt.subplots(figsize=(12, 6))
```

Comment:
Axes too small, but I suggest to remove them here entirely. There is no benefit in knowing the pixel coordinates here...